R/cite.R

```diff
@@ -0,0 +1,134 @@
+#' Cite the package and the data
+#'
+#' @param what Character vector specifying what to cite.
+#'   Can include "package", "data", "methodology_v1", "methodology_v2", or "all".
```

Comment:
So many options! It's good that users can now decide which aspect of the data to cite, 👍 to flexibility and etiquette. This is respectful to the data owners.